ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -434,24 +434,24 @@ protected LayoutParams generateLayoutParams(final ViewGroup.LayoutParams p) {
      */
     @UiThread
     private void animateFirstCardView() {
-        // Cards were in the wrong location need to reset animation each time.
-        for (int i = 0; i < getChildCount(); i++) {
-            final View child = getChildAt(i);
-            final LayoutParams lp = (LayoutParams) child.getLayoutParams();
-            switch (lp.childType) {
-                case CHILD_TYPE_CARD:
-                    bounceCardAnimation(child);
-                    return; // Stop after first Card View
+        if (mActiveCard == null) {
```

Comment:
something that I picked up from @ryancarlson which he got from Swift (or Rails) is short-circuiting conditions instead of deeply nested if blocks. What I mean by that is:
```
if (mActiveCard != null) {
    return;
}
```
at the beginning of the method so you don't have to nest the rest of the logic in an if block.

This gives the benefit of less code churn as conditionals and amount of them change over time within a method.